WooCommerce/Classes/Yosemite/DefaultStoresManager.swift

```diff
@@ -485,6 +492,34 @@ private extension DefaultStoresManager {
         sendTelemetryIfNeeded(siteID: siteID)
     }
 
+    /// Load the site with the specified URL into the session if possible.
+    ///
+    func restoreSessionSite(with url: String) {
+        let action = WordPressSiteAction.fetchSiteInfo(siteURL: url) { [weak self] result in
+            guard let self else { return }
+            switch result {
+            case .success(let site):
+                self.sessionManager.defaultSite = site
+                /// Fetch more site info if possible
+                WordPressAuthenticator.fetchSiteInfo(for: url) { [weak self] result in
```

Comment:
❓ I don't understand how this `fetchSiteInfo` is different from ours. I think it might help to explain in comments.

Reply:
Comment added in e2dc50452509a968bdfd0c30c9112a4acfa687de.